Sorts/CombSort.java

```diff
@@ -0,0 +1,64 @@
+// Java program for implementation of Comb Sort
+
+class CombSort
+{
+    // To find gap between elements
+    int getNextGap(int gap)
+    {
+        // Shrink gap by Shrink factor
+        gap = (gap*10)/13;
+        if (gap < 1)
+            return 1;
+        return gap;
+    }
+ 
+    // Function to sort arr[] using Comb Sort
+    void sort(int arr[])
```

Comment:
You can this methods declare as ```static```. If you do this you can the methods right away use in the main-method.